include/toml++/impl/preprocessor.h

```diff
@@ -366,6 +366,11 @@
 #ifdef TOML_CONFIG_HEADER
 	#include TOML_CONFIG_HEADER
 #endif
+
+// is the library being built as a shared lib/dll using meson and friends?
```

Comment:
Oh, eh. Guess this comment is technically wrong. I should have written:

```suggestion
// is the library being consumed as a shared lib/dll (using meson and friends)?
```

Reply:
SHARED_LIB is defined during both compilation and usage, so I would not consider this as completely wrong